### PR TITLE
Display all period dates

### PR DESCRIPTION
**Pull request by sylviaduarte**
hi :D

### CODE REVIEW

File: include/functions.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+function debugOutput($input){
+    echo "<pre>";
+    var_dump($input);
+    echo "</pre>";
+}
+
+
+function convertDateToMDY($dateToConvert) {
+    $datePosted = $dateToConvert;
```
Comment (ejeliazkova) [inline include/functions.php]: You don't necessarily need this line -- I would just pass $dateToConvert to DateTime() and then save it to $datePosted

File: test_db.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+include("include\initialize.php");
+$result = dbQuery("SHOW TABLES")->fetchAll();
```
Comment (ejeliazkova) [inline test_db.php]: I assume this is just for testing since you probably don't want to show your DB tables on the website

File: index.php [hunk 3]
```diff
@@ -1,111 +1,33 @@
...
-    <p class="text">If you can see this page, your new virtual host was set up successfully. Now, web content can be added and this placeholder page<sup><a href="#footnote_1">1</a></sup> should be replaced or deleted.</p>
-    <p class="text">
-        Server name: <samp><?php echo $_SERVER['SERVER_NAME']; ?></samp><br />
-        Document root: <samp><?php echo $_SERVER['DOCUMENT_ROOT']; ?></samp>
-    </p>
-    <p class="text" id="footnote_1"><small><sup>1</sup> Files: <samp>index.php</samp> and <samp>MAMP-PRO-Logo.png</samp></small></p>
-    <hr />
-    <p class="text">Diese Seite auf: <?php echo $str_available_languages; ?></p>
+      foreach ($dates as $row) {
+        echo "Started on ".convertDateToMDY($row['startDate']).". Ended on ".convertDateToMDY($row['endDate'])."<br><br>";
```
Comment (ejeliazkova) [inline index.php:19]: Always good to have a query in a function, so I'd say yes we should make this into a function that returns an array or something that you can reference whenever you want to echo something to the page in the future

**Review by ejeliazkova — CHANGES_REQUESTED** [review]
some comments 4 u

File: test_db.php (hunk 2 above)
Comment (sylviaduarte) [reply test_db.php]: yes u r right. this will go away

File: index.php (hunk 3 above)
Comment (sylviaduarte) [reply index.php:19]: muy bien gracias

**Review by ejeliazkova — APPROVED** [review]
Some functions may change as you add to your project but this looks good!

**Pull request merged**